src/quo2/components/numbered_keyboard/keyboard_key/view.cljs

```diff
@@ -0,0 +1,34 @@
+(ns quo2.components.numbered-keyboard.keyboard-key.view
+  (:require [react-native.core :as rn]
+            [quo2.theme :as quo.theme]
+            [status-im.ui.components.icons.icons :as icons]
+            [quo2.foundations.colors :as colors]
+            [quo2.components.numbered-keyboard.keyboard-key.style :as style]))
+
+(defn- get-label-color
+  [disabled theme blur]
+  (if disabled
+    (if (or (= :dark theme) blur)
+      colors/white-opa-30
+      colors/neutral-30)
+    (if (or (= :dark theme) blur)
+      colors/white
+      colors/neutral-100)))
+
+(defn- keyboard-key-internal
+  [{:keys [label? disabled theme blur? icon? on-press?]}]
```

Comment:
Change `disabled` to `disabled?`. In Clojure we suffix boolean values with `?`.

Ref: https://github.com/status-im/status-mobile/blob/develop/doc/new-guidelines.md#dont-prepend-booleans-with-is-

Is `on-press?` a boolean?

Reply:
Done